modules/testing-utils/pax-it-container/src/main/java/org/motechproject/testing/osgi/container/MotechNativeTestContainer.java

```diff
@@ -55,10 +55,12 @@
     private static final String TESTED_SYMBOLIC_NAME = "org.motechproject.testing.osgi.TestedSymbolicName";
     private static final String FAKE_MODULE_STARTUP_EVENT = "org.motechproject.testing.osgi.FakeStartupModulesEvent";
     private static final String CONTEXT_SERVICE_NAME = "org.springframework.context.service.name";
+    private static final String CONTEXT_PATH = "Context-Path";
 
     private long probeId;
     private ExamSystem examSystem;
     private boolean startupEventSent;
+    private int expectedContexts = 0;
```

Comment:
Would be better to treat this variable as the number of expected contexts, so start with 1